build.gradle

```diff
@@ -81,6 +81,8 @@ dependencies {
     implementation 'info.picocli:picocli:4.0.4'
     implementation 'org.apache.commons:commons-csv:1.7'
 
+    implementation "io.grpc:grpc-services:1.30.0"
```

Comment:
Can we somehow use the same version variable here and in https://github.com/Yelp/nrtsearch/blob/master/clientlib/build.gradle#L30 ? I think you can do it by making it a project level variable